.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/FileSystem"]
+	path = submodules/FileSystem
+	url = git@github.com:aspnet/FileSystem.git
```

Comment:
no. git and http are two transfer protocols for git. https is not considered as fall back. the url recorded here is the url used to clone the repo.